docs/labwc-config.5.scd

```diff
@@ -579,6 +580,12 @@ windows using the mouse.
 	any motion events while a keyboard is typing, and for a short while
 	after as well.
 
+*<libinput><device><clickMethod>* [none|buttonAreas|clickfinger]
+	Set the kind of click method that shall be used. Either the location of
+	where the click happens (buttonAreas), or the number of fingers
+	(clickfinger) being used determines the kind of mouse button being
+	triggered. Choose *none* if a click shall not trigger a mouse button click.
```

Comment:
```suggestion
	Configure the method by which physical clicks on a touchpad are mapped
	to mouse-button events.

		- *buttonAreas* - The touchpad is divided into distinct regions
		corresponding to left, middle or right buttons; clicking within
		each region will trigger the corresponding event.

		- *clickfinger* - Clicking with one, two or three fingers will
		produce left, middle or right button events without regard to
		the location of a click. The specific map from finger count to
		button is infuenced by the *tapButtonMap* option.

		- *none* - Physical clicks will not produce button events.

	The default method depends on the touchpad hardware.
```

Reply:
I think the rational for the different names are that:
* "button areas" is a valid english term.
* "clickfinger" however, is a newly invented "name" and therefore a single word.

Reply:
I improved the documentation similar to what you proposed. However, I added more details about the behavior of the "buttonAreas" option.

And I removed the 
> The specific map from finger count to button is influenced by the *tapButtonMap* option.

This behavior is not documented in the libinput docs. Additionally, I tried it out, and it does not seem to be the case that the * tapButtonMap*  option influences Clickfinger.

I also changed the exact formatting to be more consisting with the already existing bullet point list.

Reply:
- [ ] Based on the GitHub formatting of *.scd* I think something is still messed up with the formatting.

Reply:
My change made it a bit better. But the GitHub rendering of this file seems to be very busted in any case.